config.yaml

```
@@ -55,4 +55,13 @@ data_sources:
-      timeout: 30
\ No newline at end of file
+      timeout: 30
+
+  - name: jartic
+    type: jartic
+    enabled: false
+    config:
+      base_url: https://www.compusophia.com/jartic_data
+      timeout: 300
+      cache_dir: data/jartic/cache
+      cleanup_after_parse: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)